### PR TITLE
Add contact investigation

### PR DESCRIPTION
**Pull request by ODORA0**

### CODE REVIEW

**Comment by ODORA0** [comment]
@pirupius FYA

File: src/views/contact-investigation/contact-investigation.tsx [hunk 1]
```diff
@@ -1,15 +1,83 @@
...
 const ContactInvestigation: React.FC<PatientChartProps> = ({ patientUuid }) => {
   const { t } = useTranslation();
   const title = t("contactInvestigation", "Contact Investigation");
 
-  return <EmptyStateComingSoon headerTitle={title} displayText={title} />;
+  const columns: EncounterListColumn[] = useMemo(
+    () => [
+      {
+        key: "days",
+        header: t("contact_id_number", "Contact ID number"),
```
Comment (pirupius) [inline src/views/contact-investigation/contact-investigation.tsx:25]: change these keys to `camelCase`

File: translations/en.json [hunk 2]
```diff
@@ -20,5 +20,8 @@
   "suspectedCases": "Suspected cases",
   "statusOfContact": "Status of contact",
   "totalCases": "Total cases",
-  "totalDeaths": "Total deaths"
+  "totalDeaths": "Total deaths",
+  "contact_id_number": "Contact ID number",
+  "prolonged_contact": "Prolonged Contact",
+  "type_of_contact": "Type of Contact"
```
Comment (pirupius) [inline translations/en.json:26]: let's main `camelCase` for these

**Review by pirupius — CHANGES_REQUESTED** [review]

File: src/views/contact-investigation/contact-investigation.tsx (hunk 1 above)
Comment (ODORA0) [reply src/views/contact-investigation/contact-investigation.tsx]: @pirupius Done

File: translations/en.json (hunk 2 above)
Comment (ODORA0) [reply translations/en.json]: @pirupius Done

**Review by pirupius — APPROVED** [review]

**Pull request merged**